src/main/java/uk/gov/justice/plugin/validator/AnnotationValidator.java

```diff
@@ -8,4 +8,6 @@
 public interface AnnotationValidator {
 
     ValidationResult validate(final Class<?> classWithAnnotation, final Map<String, String> validationProperties) throws ValidationException;
+
+    String worksOnAnnotatedClass();
```

Comment:
not sure if there is a better name than this.
Something like appliesToAnnotation maybe? not sure...

Reply:
@matt-rich will change it to what you suggest..